weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx

```diff
@@ -537,6 +538,11 @@ const CenterProjectTablesBrowser: React.FC<
       'updated at': moment.utc(b.updatedAt).local().calendar(),
       'created at': moment.utc(b.createdAt).local().calendar(),
       'created by': b.createdByUserName,
+      // we use the # of steps in the run history to compute rows quickly. This works for stream tables
+      // because the table leverages run history. For logged tables, we must open to table in order to
+      // to know the number of rows which is a much more expensive operation. Instead, we will just
```

Comment:
you have "to" at the end of the previous line

Reply:
womp, i always self review my code and never my comments :P fixed